src/Arcus.BackgroundJobs.Tests.Integration/Jobs/DatabricksJobMetricsTests.cs

```diff
@@ -1,122 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-using System.Threading.Tasks;
-using Arcus.BackgroundJobs.Databricks;
-using Arcus.BackgroundJobs.Tests.Integration.Hosting;
-using Arcus.Security.Core;
-using Microsoft.Azure.Databricks.Client;
-using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
-using Moq;
-using Polly;
-using Xunit;
-using Xunit.Abstractions;
-using Xunit.Sdk;
-
-namespace Arcus.BackgroundJobs.Tests.Integration.Jobs
```

Comment:
Feel free to re-introduce

Reply:
Hmm, seems like our test job on Databricks is deleted? 😢

Reply:
I still see `RestTriggerTest` job

Reply:
Ah, okay. Checked with a created job and it still works (oef).